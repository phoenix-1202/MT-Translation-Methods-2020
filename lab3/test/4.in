funWithIf :: Int -> Int -> Int
funWithIf a b | a == 0 || b == 0 = a + b
funWithIf a b = if a > b then (funWithIf (a `mod` b) b) else (funWithIf a b `mod` a)