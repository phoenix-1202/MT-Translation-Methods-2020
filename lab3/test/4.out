#include <bits/stdc++.h>;
using namespace std;

int funWithIf(int arg1, int arg2) {
if ((arg1 == 0) || (arg2 == 0)) {
return (arg1 + arg2);
}
if ((arg1 > arg2)) {
return funWithIf((arg1 % arg2), arg2);
}
else {
return funWithIf(arg1, (arg2 % arg1));
}
}

//EXIT